### PR TITLE
Documented default pod and service subnets

### PR DESCRIPTION
**Pull request by RijulGulati**
Added default CIDR for pod subnet and service subnet.
This PR is an attempt to resolve issue #3117

### CODE REVIEW

File: site/content/docs/user/configuration.md [hunk 1]
```diff
@@ -181,6 +181,8 @@ networking:
   podSubnet: "10.244.0.0/16"
 {{< /codeFromInline >}}
 
+By default, kind uses ```10.244.0.0/16``` pod subnet for IPv4 and ```fd00:10:244::/56``` pod subnet for IPv6.
```
Comment (aojea) [inline site/content/docs/user/configuration.md:184]:
```suggestion
By default, `kind` uses ```10.244.0.0/16``` pod subnet for IPv4 and ```fd00:10:244::/56``` pod subnet for IPv6.
```

@BenTheElder single ticks?

Comment (RijulGulati) [reply site/content/docs/user/configuration.md:184]: I had thought of highlighting `kind` as well, but I saw kind being used without highlights in other places. So left it as it is.

Comment (aojea) [reply site/content/docs/user/configuration.md:184]: Ben is the best person to decide, but I think ticks is better

**Comment by aojea** [comment]
/lgtm
/approve

those were nits, this moves to a better state anyway

File: site/content/docs/user/configuration.md (hunk 1 above)
Comment (BenTheElder) [reply site/content/docs/user/configuration.md:184]: kind or KIND but not KinD
`kind` or `KIND` but not KinD

:-)

thanks for documenting this, been a hectic period with https://kubernetes.io/blog/2023/03/10/image-registry-redirect/, the docker hub announcements, the runc cves ...

Comment (BenTheElder) [reply site/content/docs/user/configuration.md:184]: @aojea we could add a note to the contributing guide under docs development

**Pull request merged**